CHANGELOG.md

```diff
@@ -102,8 +103,10 @@ Changes recorded below are for all of the package versions in the list:
 - Added new governance related data types to Conway: #3216
   - `GovernanceActionState`
   - `GovernanceProcedure`
+  - `GovernanceMetadata`
   - `TallyEnv`
-
+  - `RatifyEnv`
+  - `RatifySignal`
```

Comment:
All of this changes now need into the `era/conway/impl/CHANGELOG.md` file, since we switched to PVP for versioning

Reply:
Could you please remove the changes in this file?

This changelog is fore previous release process and we should not change it